backend/src/main/java/com/skr/virtuallibrary/repositories/BookListRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.skr.virtuallibrary.repositories;
+
+import com.skr.virtuallibrary.entities.BookList;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+import java.util.List;
+
+public interface BookListRepository extends MongoRepository<BookList, String> {
+
+    List<BookList> findAllByUserId(String userId);
+
+    List<BookList> findAllByNameAndUserId(String name, String userId);
```

Comment:
Jeśli nazwa listy jest unikalna, to czemu metoda zwraca listę?

Reply:
Bo nie jest unikalna

Reply:
poprawione